app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/action/AndroidAction.kt

```diff
@@ -0,0 +1,9 @@
+package org.jellyfin.androidtv.ui.playback.overlay.action
+
+import org.jellyfin.androidtv.ui.playback.overlay.VideoPlayerAdapter
+
+interface AndroidAction {
```

Comment:
Would it be possible to also inherit this class from our CustomAction (perhaps merge the two?) so we don't have 2 separate ways to trigger them.

Reply:
Since the default Actions need to inherit from a Android base class, and Kotlin doesn't support multiple inheritance, I've restructured it to:

```
CustomAction (interface) -> CustomActionBase (base class) -> Action*
```

Reply:
Not easily, doing some further digging, we need to have a way to call `onCustomActionClicked` if we get Android actions to use the CustomAction class

Since we can't do multiple inheritance, we either have to inherit the default action (won't derive `onCustomActionClicked`), or reimplement the Android component we no longer derive from

I'm sure there's a clever way to do this, but because we need the view for CustomActions it means they fundamentally require different things